### PR TITLE
implements fetchAllUsers function

### PR DESCRIPTION
**Pull request by waahm7**
Implements the getUsers function in DataStore class to fetch all users that have sent at least one message in DataStore class. 

- [x] step 3 of community page

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -23,9 +23,8 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
+
+import java.util.*;
```
Comment (raufrajar) [inline src/main/java/com/google/codeu/data/Datastore.java]: Preferable to use individual imports.

File: src/main/java/com/google/codeu/servlets/UserListServlet.java [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+    public void init() {
+        datastore = new Datastore();
+    }
 
-        response.getOutputStream().println("this will be my user list");
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        Set<String> users=datastore.getUsers();
```
Comment (raufrajar) [inline src/main/java/com/google/codeu/servlets/UserListServlet.java]: nit: You don't need users, could just do `for (String user:datastore.getUsers())`

**Review by raufrajar — APPROVED** [review]

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (waahm7) [reply src/main/java/com/google/codeu/data/Datastore.java]: Thanks! Fixed it.

File: src/main/java/com/google/codeu/servlets/UserListServlet.java (hunk 2 above)
Comment (waahm7) [reply src/main/java/com/google/codeu/servlets/UserListServlet.java]: Thanks! Changed this too.

**Pull request merged**